### DISCUSSION TITLE
What happens to spring-cloud/spring-cloud-aws

### DISCUSSION THREAD

**Discussion by arunkpatra** [awspring/spring-cloud-aws#61] [General]
Just wanted to get some clarity what's the path forward for the spring-cloud/spring-cloud-aws repo. Almost all google search traffic lands up on that repo. Is there a clear marketing communication somewhere. I just saw @maciejwalkowiak's tweet.

**Comment by MatejNedic** [comment]
Hey @arunkpatra,
we are still discussing how to communicate future releases and other stuff (we have ideas already just need to align them). That doesn't mean we don't accept new ideas! Currently focus is on 3.0 road map in context that we want to bring repo up to the point where it will be easier for people to contribute and add new integrations/features. So until we bring quality of 3.0 up to standard we believe it should be, we aren't focusing so much on marketing/communication. 

EDIT: 
As for [spring-cloud-aws](https://github.com/spring-cloud/spring-cloud-aws) repository it is there since branch 2.2.x is still maintained bug fixes and so on. In readme of old repository it can be found on top that 2.2.x is maintained and all future development has been moved to this repo.

**Reply by arunkpatra** [reply]
Thanks @MatejNedic , that makes sense. Looked at PR #56 ... I guess you guys have already started :-) Happy to jump on. Let me know when you guys are ready
